storage/pipeline/input.go

```diff
@@ -341,14 +350,25 @@ func (m *Sealing) SectorAddPieceToAny(ctx context.Context, size abi.UnpaddedPiec
 	return api.SectorOffset{Sector: res.sn, Offset: res.offset.Padded()}, res.err
 }
 
+func (m *Sealing) getClaimTerms(ctx context.Context, deal api.PieceDealInfo) (pieceClaimBounds, error) {
+	// TODO: TODO! TODO! get the real claim bounds here
+
+	return pieceClaimBounds{
+		claimTermEnd: deal.DealProposal.EndEpoch + policy.GetSectorMaxLifetime(abi.RegisteredSealProof_StackedDrg32GiBV1_1, network.Version17),
+	}, nil
+}
```

Comment:
Only todo, needs some API plumbing, cc @geoff-vball

Reply:
API is in the latest version of `gstuart/integrate-verifreg-changes-fip45`. The api you'll want is `StateGetAllocationForPendingDeal`